rider/migrations/0001_initial.py

```diff
@@ -9,23 +9,23 @@ class Migration(SchemaMigration):
 
     def forwards(self, orm):
         # Adding model 'Rider'
-        db.create_table('rider_rider', (
+        db.create_table(u'rider_rider', (
```

Comment:
it must have become necessary at some point in the past as this was the difference between migration 1 and 2